jar-service/src/main/java/com/sixsq/slipstream/application/RootApplication.java

```diff
@@ -231,19 +251,13 @@ private Directory attachDownloadsDirectory() {
 		return directory;
 	}
 
-	private void attachReports(RootRouter router)
-			throws ConfigurationException, ValidationException {
+	private void attachReports(RootRouter router) throws ConfigurationException, ValidationException {
 		router.attach("/reports", new ReportRouter(getContext(), router.getApplication()));
 	}
```

Comment:
Shouldn't attachReports() follow the introduced pattern and use guardAndAttach() as well?